ktor-utils/posix/src/io/ktor/util/CryptoNative.kt

```diff
@@ -23,4 +23,6 @@ public actual fun Digest(name: String): Digest = error("[Digest] is not supporte
 /**
  * Compute SHA-1 hash for the specified [bytes]
  */
-public actual fun sha1(bytes: ByteArray): ByteArray = Sha1().digest()
+public actual fun sha1(bytes: ByteArray): ByteArray = Sha1().apply {
```

Comment:
`digest(bytes)` can be used here https://github.com/ktorio/ktor/blob/b1ab7659065ee0013a57210df77027677489de4e/ktor-utils/common/src/io/ktor/util/HashFunction.kt#L12)